packages/application-generic/src/services/distributed-lock/distributed-lock.service.spec.ts

```diff
@@ -9,41 +6,51 @@ import { DistributedLockService } from './distributed-lock.service';
 
 const distributedLockService = new DistributedLockService();
 let redisClient: Redis;
-let spyDecreaseLockCounter: SinonSpy;
-let spyIncreaseLockCounter: SinonSpy;
-let spyLock: SinonSpy;
-let spyUnlock: SinonSpy;
+let spyDecreaseLockCounter: jest.SpyInstance;
```

Comment:
had to convert some tests to the jest

Reply:
the `application-generic` package is using Jest for running tests...

and I'm a Jest guy :P
![Screenshot 2023-04-04 at 11 25 07](https://user-images.githubusercontent.com/2607232/229748610-6f2ef977-1950-42da-9ed1-a4be8449b1ba.png)